core/theme/Theme.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace luya\theme;
+
+use luya\helpers\Json;
+use luya\helpers\StringHelper;
+use Yii;
+use yii\base\BaseObject;
+use yii\base\InvalidConfigException;
+
+/**
+ * Extend the yii2 theme component and build up the `$pathMap` depends on given \luya\theme\ThemeConfig which include parents inheritance.
+ *
+ * @see \luya\theme\ThemeConfig
+ * @author Bennet Klarhoelter <boehsermoe@me.com>
```

Comment:
We try to arrange php doc always the same way. author and since are the last properties while @property habe a line break between the common tags, see the example: https://github.com/luyadev/luya/blob/master/docs/guide/luya-guideline.md#phpdoc